hist/hist/inc/TH2Poly.h

```diff
@@ -130,6 +130,9 @@ class TH2Poly : public TH2 {
 
    //functions not to be used for TH2Poly
 
+   void AddBinContent(Int_t) override { return; }           ///< NOT IMPLEMENTED for TH2Poly
+   void AddBinContent(Int_t, Double_t) override { return; } ///< NOT IMPLEMENTED for TH2Poly
```

Comment:
I would consider adding a `static_assert` to inform the user

Reply:
The `static_assert` would trigger whether the function are used or not.  Consider using a call `Fatal`.

Reply:
I have now used `Error`, because this is already done for many of the methods at the end of `TH2Poly.cxx`.